### PR TITLE
Add documentation on how to convert F3Net model

### PR DESCRIPTION
**Pull request by achetver**
### Details:
 - Add instruction to convert [F3Net](https://github.com/weijun88/F3Net) model from PyTorch to ONNX format.

### Tickets:
 - 42574

### CODE REVIEW

File: docs/MO_DG/prepare_model/convert_model/onnx_specific/Convert_F3Net.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Convert PyTorch* F3Net to the Intermediate Representation {#openvino_docs_MO_DG_prepare_model_convert_model_onnx_specific_Convert_F3Net}
+
+[F3Net](https://github.com/weijun88/F3Net): Fusion, Feedback and Focus for Salient Object Detection.
```
Comment (lazarevevgeny) [inline docs/MO_DG/prepare_model/convert_model/onnx_specific/Convert_F3Net.md:3]: Please, add a hash commit which we know the model works for sure.

**Review by lazarevevgeny — CHANGES_REQUESTED** [review]

File: docs/MO_DG/prepare_model/convert_model/onnx_specific/Convert_F3Net.md (hunk 1 above)
Comment (achetver) [reply docs/MO_DG/prepare_model/convert_model/onnx_specific/Convert_F3Net.md:3]: Done.

**Review by vgavrilo — APPROVED** [review]

File: docs/MO_DG/prepare_model/convert_model/onnx_specific/Convert_F3Net.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+## Download the F3Net Model
```
Comment (lazarevevgeny) [inline docs/MO_DG/prepare_model/convert_model/onnx_specific/Convert_F3Net.md:5]: Download and Convert the Model to ONNX\*

File: docs/MO_DG/prepare_model/convert_model/onnx_specific/Convert_F3Net.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+cfg = Config(mode='test', snapshot=<path_to_checkpoint_dir>)
+net = F3Net(cfg)
+image = torch.zeros([1, 3, 352, 352])
+torch.onnx.export(net, image, 'f3net.onnx', export_params=True, do_constant_folding=True, opset_version=11)
+```
+This code produces ONNX* model file `f3net.onnx`. It was tested on commit eecace3adf1e8946b571a4f4397681252f9dc1b8 on
+master branch.
+
+## Convert ONNX* F3Net model to IR
```
Comment (lazarevevgeny) [inline docs/MO_DG/prepare_model/convert_model/onnx_specific/Convert_F3Net.md:24]: Model

File: docs/MO_DG/prepare_model/convert_model/onnx_specific/Convert_F3Net.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
+# Convert PyTorch* F3Net to the Intermediate Representation {#openvino_docs_MO_DG_prepare_model_convert_model_onnx_specific_Convert_F3Net}
+
+[F3Net](https://github.com/weijun88/F3Net): Fusion, Feedback and Focus for Salient Object Detection.
+
+## Download the F3Net Model
+
+To download the pre-trained model or train model by yourself, refer to the 
+[instruction](https://github.com/weijun88/F3Net/blob/master/README.md) in F3Net model repository. 
+To obtain F3Net in ONNX* format you need to put script with next code in `src` folder of model repository and run it:
```
Comment (lazarevevgeny) [inline docs/MO_DG/prepare_model/convert_model/onnx_specific/Convert_F3Net.md:9]: `To download the pre-trained model or train the model by yourself, refer to the 
[instruction](https://github.com/weijun88/F3Net/blob/master/README.md) in the F3Net model repository. 
Firstly, the model should be converted to ONNX\* format. Create and run the script with the following content in the `src` directory of the model repository:

File: docs/MO_DG/prepare_model/convert_model/onnx_specific/Convert_F3Net.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+from dataset import Config
+from net import F3Net
+
+cfg = Config(mode='test', snapshot=<path_to_checkpoint_dir>)
+net = F3Net(cfg)
+image = torch.zeros([1, 3, 352, 352])
+torch.onnx.export(net, image, 'f3net.onnx', export_params=True, do_constant_folding=True, opset_version=11)
+```
+This code produces ONNX* model file `f3net.onnx`. It was tested on commit eecace3adf1e8946b571a4f4397681252f9dc1b8 on
```
Comment (lazarevevgeny) [inline docs/MO_DG/prepare_model/convert_model/onnx_specific/Convert_F3Net.md:21]: The script generates the ONNX\* model file `f3net.onnx`. The model conversion was tested with repository hash commit `eecace3adf1e8946b571a4f4397681252f9dc1b8`.

**Review by lazarevevgeny — CHANGES_REQUESTED** [review]

File: docs/MO_DG/prepare_model/convert_model/onnx_specific/Convert_F3Net.md (hunk 2 above)
Comment (achetver) [reply docs/MO_DG/prepare_model/convert_model/onnx_specific/Convert_F3Net.md]: Done.

File: docs/MO_DG/prepare_model/convert_model/onnx_specific/Convert_F3Net.md (hunk 3 above)
Comment (achetver) [reply docs/MO_DG/prepare_model/convert_model/onnx_specific/Convert_F3Net.md]: Done.

File: docs/MO_DG/prepare_model/convert_model/onnx_specific/Convert_F3Net.md (hunk 4 above)
Comment (achetver) [reply docs/MO_DG/prepare_model/convert_model/onnx_specific/Convert_F3Net.md]: Done.

File: docs/MO_DG/prepare_model/convert_model/onnx_specific/Convert_F3Net.md (hunk 5 above)
Comment (achetver) [reply docs/MO_DG/prepare_model/convert_model/onnx_specific/Convert_F3Net.md]: Done.

**Comment by achetver** [comment]
@mvafin @yekruglov please, take a look at this PR.

**Review by mvafin — APPROVED** [review]

**Review by lazarevevgeny — APPROVED** [review]

**Review by yekruglov — APPROVED** [review]

File: docs/MO_DG/prepare_model/convert_model/onnx_specific/Convert_F3Net.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
+# Convert PyTorch* F3Net to the Intermediate Representation {#openvino_docs_MO_DG_prepare_model_convert_model_onnx_specific_Convert_F3Net}
+
+[F3Net](https://github.com/weijun88/F3Net): Fusion, Feedback and Focus for Salient Object Detection.
```
Comment (tsavina) [inline docs/MO_DG/prepare_model/convert_model/onnx_specific/Convert_F3Net.md:3]:
```suggestion
[F3Net](https://github.com/weijun88/F3Net): Fusion, Feedback and Focus for Salient Object Detection
```

File: docs/MO_DG/prepare_model/convert_model/onnx_specific/Convert_F3Net.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+
+from dataset import Config
+from net import F3Net
+
+cfg = Config(mode='test', snapshot=<path_to_checkpoint_dir>)
+net = F3Net(cfg)
+image = torch.zeros([1, 3, 352, 352])
+torch.onnx.export(net, image, 'f3net.onnx', export_params=True, do_constant_folding=True, opset_version=11)
+```
+The script generates the ONNX* model file f3net.onnx. The model conversion was tested with repository hash commit eecace3adf1e8946b571a4f4397681252f9dc1b8.
```
Comment (tsavina) [inline docs/MO_DG/prepare_model/convert_model/onnx_specific/Convert_F3Net.md:22]:
```suggestion
The script generates the ONNX\* model file f3net.onnx. The model conversion was tested with the repository hash commit `eecace3adf1e8946b571a4f4397681252f9dc1b8`.
```

**Review by tsavina — CHANGES_REQUESTED** [review]

File: docs/MO_DG/prepare_model/convert_model/onnx_specific/Convert_F3Net.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
+# Convert PyTorch* F3Net to the Intermediate Representation {#openvino_docs_MO_DG_prepare_model_convert_model_onnx_specific_Convert_F3Net}
+
+[F3Net](https://github.com/weijun88/F3Net): Fusion, Feedback and Focus for Salient Object Detection.
+
+## Download and Convert the Model to ONNX*
+
+To download the pre-trained model or train the model by yourself, refer to the
```
[the remark this answers is not in the document]
Comment (achetver) [reply docs/MO_DG/prepare_model/convert_model/onnx_specific/Convert_F3Net.md]: Done.

File: docs/MO_DG/prepare_model/convert_model/onnx_specific/Convert_F3Net.md [hunk 9]
```diff
@@ -0,0 +1,28 @@
+# Convert PyTorch* F3Net to the Intermediate Representation {#openvino_docs_MO_DG_prepare_model_convert_model_onnx_specific_Convert_F3Net}
+
+[F3Net](https://github.com/weijun88/F3Net): Fusion, Feedback and Focus for Salient Object Detection.
+
+## Download and Convert the Model to ONNX*
+
+To download the pre-trained model or train the model by yourself, refer to the 
+[instruction](https://github.com/weijun88/F3Net/blob/master/README.md) in the F3Net model repository. Firstly, 
+the model should be converted to ONNX\* format. Create and run the script with the following content in the `src`
```
[the remark this answers is not in the document]
Comment (achetver) [reply docs/MO_DG/prepare_model/convert_model/onnx_specific/Convert_F3Net.md]: Done.

File: docs/MO_DG/prepare_model/convert_model/onnx_specific/Convert_F3Net.md (hunk 6 above)
Comment (achetver) [reply docs/MO_DG/prepare_model/convert_model/onnx_specific/Convert_F3Net.md]: Done.

File: docs/MO_DG/prepare_model/convert_model/onnx_specific/Convert_F3Net.md (hunk 7 above)
Comment (achetver) [reply docs/MO_DG/prepare_model/convert_model/onnx_specific/Convert_F3Net.md]: Done.

**Review by tsavina — APPROVED** [review]

**Review by andrew-zaytsev — APPROVED** [review]

**Pull request merged**